doc/generic/pgf/text-en/pgfmanual-en-tikz-graphs.tex

```diff
@@ -243,8 +243,9 @@ \subsubsection{Concept: Chain Groups}
 
 As can be seen, the placement is not particularly nice by default, use the
 algorithms from the graph drawing libraries to get a better layout. For
-instance, adding |tree layout| to the above code results in the following
-somewhat more pleasing rendering:
+instance, adding |tree layout| to the above code (and
```

Comment:
So this also closes #881 ?

Reply:
@ilayn, sorry, I thought that I had commented on that already, but it seems that didn't work. So here "again".

You are absolutely right. This is, because I did that change in my forks master branch before following your suggestions to do new changes only in branches. A day after doing these changes I thought that I maybe could have avoided that by checking out the commit before I made that change and creating a branch from there.(?)

I hope that I don't have to do everything again, because this is hopefully a trivial change and therefore doesn't mess up anything when this is applied again.

Reply:
No problem I'll edit this issue text and they will all be closed automagically once this is merged. I think you fought well with git.